### PR TITLE
Steps for variables and fragments

### LINKED ISSUE #6: Show how to use graphql variables and fragments
As titled

### PR DESCRIPTION
**Pull request by sameer-coder**
This PR includes:

- Variables step
- Fragments step
- Deck updates
- Tests coverage 💯 

Closes #6

### CODE REVIEW

**Comment by ja-bravo** [comment]
Looks good to me, all 3 steps run, pass the tests, and you can copy/paste the Trying it out curl command and get the appropriate result.

The only issue I could find is that ```Step 8: Trying it out``` slide doesn't fit on my screen (MacBook pro 15) so the topPosts query result gets cut

**Comment by sameer-coder** [comment]
> Looks good to me, all 3 steps run, pass the tests, and you can copy/paste the Trying it out curl command and get the appropriate result.
> 
> The only issue I could find is that the ```Step 8: Trying it out``` slide doesn't fit on my screen (MacBook pro 15) so the topPosts query result gets cut

Thanks for the review. I will adjust that slide so that it fits

**Comment by simoneb** [comment]
@ja-bravo it looks like you left a plain comment rather than a review. To review a PR go to the "files changed" tab, then click "review changes", then submit your review (comment/approve/request changes). Because most repos require an approving review, if you don't submit one the author won't be able to merge.

**Review by ja-bravo — APPROVED** [review]

File: deck.md [hunk 1]
```diff
@@ -753,6 +753,330 @@ curl --request POST \
...
+  --url http://localhost:4000/graphql \
+  --header 'Content-Type: application/json' \
+  --data '{"query":"{ me { name, numberOfPosts } }"}'
+```
+
+```bash
+curl --request POST \
+  --url http://localhost:4000/graphql \
+  --header 'Content-Type: application/json' \
+  --data '{"query":"{ \n  topPosts(count: 1) {\n    title\n  }\n}"}'
```
Comment (giacomorebonato) [inline deck.md:864]: nitpick: escape sequences can be removed

File: postgrator.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "migrationPattern": "migrations/*",
   "driver": "pg",
   "host": "127.0.0.1",
-  "port": 5432,
+  "port": 5433,
```
Comment (giacomorebonato) [inline postgrator.json:5]: thought: maybe out of context for the workshop, but I noticed these deprecation warnings from Postgrator:

```bash
postgrator deprecated Config option "port". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "host". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "username". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "password". Implement execQuery function instead. node_modules/postgrator-cli/postgrator-cli.js:221:26
postgrator deprecated Config option "migrationDirectory". use "migrationPattern" instead using glob match. e.g. path.join(__dirname, '/migrations/*') node_modules/postgrator-cli/postgrator-cli.js:221:26
```

Comment (simoneb) [reply postgrator.json:5]: yeah good catch, postgrator's API has changed, let's reflect those changes here as well

**Review by giacomorebonato — APPROVED** [review]
Looking good! 💥

File: postgrator.json (hunk 2 above)
Comment (sameer-coder) [reply postgrator.json:5]: We are using `postgrator-cli` latest version and not `postgrator` directly. 
`postgrator-cli` is still using postgrator 4.0. I don't get any warnings with the current version.
@simoneb Fastify workshop also uses the same version of `postgrator-cli`. Should I leave it for now till they upgrade or do you want me to use postgrator 5.0 directly?

Comment (simoneb) [reply postgrator.json:5]: we're good then 👌

**Pull request merged**